admin/class-create-theme.php

```diff
@@ -50,8 +50,16 @@ function create_block_theme_enqueue() {
 			$asset_file['dependencies'],
 			$asset_file['version']
 		);
+		wp_enqueue_style(
+			'create-block-theme-styles',
+			plugins_url( 'build/plugin-sidebar.css', dirname( __FILE__ ) ),
+			false,
+			$asset_file['version'],
+			'all'
+		);
 		wp_enqueue_script(
 			'create-block-theme-slot-fill',
+			'create-block-theme-styles'
```

Comment:
Isn't this unnecessary? What is newly enqueued in this PR is a stylesheet, not a script.

Reply:
Nice catch. Thank you.